### PR TITLE
inform users about lag

### PR DESCRIPTION
**Pull request by mfranczy**
inform users that they have to wait to see result after migrate cmd

### CODE REVIEW

**Comment by vmindru** [comment]
+1

**Comment by vmindru** [comment]
let's run test against this and the merge.

**Comment by vmindru** [comment]
*then

**Comment by mfranczy** [comment]
Jenkins: ok to test

**Comment by ncoghlan** [comment]
Jenkins test

File: cockpit/leapp.html [hunk 1]
```diff
@@ -301,7 +306,11 @@
             //output.empty();
             cmd_stdout(data);
             output.append(document.createTextNode("Command completed successfully"));
+            if (wait_for_result_msg) {
+                output.append(document.createTextNode("\n\nPlease wait about 2 minutes to see results.."));
```
Comment (vmindru) [inline cockpit/leapp.html]: agree.

**Pull request merged**